docs/content/config/advanced/maintenance/update-and-cleanup.md

```diff
@@ -23,19 +23,6 @@ For more details, see the [manual](https://containrrr.github.io/watchtower/)
 
 ## Automatic Cleanup
 
-When you are pulling new images in automatically, it would be nice to have them cleaned up as well. There is also a docker image for this: [`spotify/docker-gc`](https://hub.docker.com/r/spotify/docker-gc/).
+For cleanup you can use the [`--cleanup`](https://containrrr.github.io/watchtower/arguments/#cleanup) option provided by `containrrr/watchtower`.
 
-A Docker Compose example:
-
-```yaml
-services:
-  docker-gc:
-    restart: always
-    image: spotify/docker-gc:latest
-    volumes:
-      - /var/run/docker.sock:/var/run/docker.sock
-```
-
-For more details, see the [manual](https://github.com/spotify/docker-gc/blob/master/README.md)
-
-Or you can just use the [`--cleanup`](https://containrrr.github.io/watchtower/arguments/#cleanup) option provided by `containrrr/watchtower`.
+Or consider using ['docker prune command'](https://docs.docker.com/engine/reference/commandline/system_prune/)
```

Comment:
A little more context. `docker image prune` is probably sufficient here than `docker system prune`. Clarified that this isn't automatic / scheduled, but manually invoked.

```suggestion
!!! tip "Manual cleanup"

    While `watchtower` also supports running once with `docker run` or `compose.yaml` with the `--run-once` option, you can also directly invoke cleanup with [`docker image prune --all`][docker-docs-prune-image] or the more thorough [`docker system prune --all`][docker-docs-prune-system] (_also removes unused containers, networks, build cache_). Avoid the `--all` option if you only want to cleanup "dangling" content.

[docker-docs-prune-image]: https://docs.docker.com/engine/reference/commandline/image_prune/
[docker-docs-prune-system]: https://docs.docker.com/engine/reference/commandline/system_prune/
```